frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/HealthCard.tsx

```diff
@@ -70,27 +71,24 @@ const HealthCard: React.FC<HealthCardProps> = ({
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>Health</DashboardCardTitle>
+        <DashboardCardTitle>Status</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody>
         <HealthBody>
```

Comment:
`HealthBody` has to be imported from `'@console/shared/src/components/dashboard/status-card/HealthBody';` otherwise there is margin problem on the status. (This one was a tough one to find out)

Reply:
Yes, I learned that the hard way. I didn't see your comments until now for some reason :(